test/datapipes/test_healpix_couple.py

```diff
@@ -86,6 +89,8 @@ def scaling_double_dict():
         "z1000": {"mean": 0, "std": 2},
         "z250": {"mean": 0, "std": 2},
         "z500": {"mean": 0, "std": 2},
+        "lsm": {"mean": 0, "std": 2},
```

Comment:
@ivanauyeung do you understand why all these channels use 
`{"mean": 0, "std": 2},` and not `{"mean": 0, "std": 1},`
if the goal here is to provide defaults that make no change to the data and the scaling is 
`var = (var-mean)/std`
than `{"mean": 0, "std": 1},` would be the right choice

Reply:
This definition is copied directly from test/datapipes/test_healpix.py. So I also have no idea about why it is 2. test_healpix_couple.py is used for ci test only and it will not affect training or inference.